cumulus_lambda_functions/daac_archiver/daac_archiver_logic.py

```diff
@@ -117,7 +117,7 @@ def send_to_daac_internal(self, uds_cnm_json: dict):
                 },
                 "identifier": uds_cnm_json['identifier'],
                 "submissionTime": f'{TimeUtils.get_current_time()}Z',
-                "provider": granule_identifier.tenant,
+                "provider": daac_config['daac_provider'],
```

Comment:
@wphyojpl can we fallback to granule_identifier.tenant if daac_provider isn't set?

Reply:
does that mean daac_provider is optional during the creation of daac_config?